jobs/furnishings/src/furnishings/stage_processors/stage_one.py

```diff
@@ -41,33 +44,85 @@ async def process(self, batch_processing: BatchProcessing):
                 business_id=batch_processing.business_id
                 )
         if not furnishings:
+            # send first notification if no furnishing entry exists
             await self._send_first_round_notification(batch_processing)
         else:
             # send paper letter if business is still not in good standing after 5 days of email letter sent out
-            pass
+            valid_furnishing_names = [
+                Furnishing.FurnishingName.DISSOLUTION_COMMENCEMENT_NO_AR,
+                Furnishing.FurnishingName.DISSOLUTION_COMMENCEMENT_NO_TR,
+                Furnishing.FurnishingName.DISSOLUTION_COMMENCEMENT_NO_AR_XPRO,
+                Furnishing.FurnishingName.DISSOLUTION_COMMENCEMENT_NO_TR_XPRO
+            ]
+            tz = pytz.timezone('UTC')
+            today_date = tz.localize(datetime.today())
```

Comment:
`furnishing.created_date` is a UTC timezone, so we need to compare to today in UTC.